### PR TITLE
#11620 Release 22.8.0

### LINKED ISSUE #11620: Release 22.8.0
This is the ticket to track the release of 22.8.0

### PR DESCRIPTION
**Pull request by adiroiban**
## Scope and purpose

Fixes #11620

This is the release branch for 22.8.0

Docs at https://twisted--11622.org.readthedocs.build/en/11622/
Release candidate at https://pypi.org/project/Twisted/22.8.0rc1/

### CODE REVIEW

**Comment by adiroiban** [comment]
this is ready. Please review. Thanks!

**Comment by adiroiban** [comment]
please review

File: NEWS.rst
Comment (exarkun) [inline NEWS.rst]: Huh... `#0`?

Comment (adiroiban) [reply NEWS.rst]: I will remove it.
I think that was for some work related to Trac to GitHub.

File: NEWS.rst
Comment (adiroiban) [inline NEWS.rst:65]: This should be reverted and removed from the release

see https://github.com/twisted/twisted/issues/10286

Comment (adiroiban) [reply NEWS.rst:65]: It looks like we have a different issue.
So this can go and is not a release blocker.

Separate ticket at https://github.com/twisted/twisted/issues/11626

**Comment by adiroiban** [comment]
Thanks @exarkun for the review.

This is ready to be reviewed by @twisted/twisted-contributors 

As soon as I get an approval for the PR, I will create the release tag , and from there the automated release process should do its magic.

needs-review

File: NEWS.rst
Comment (glyph) [inline NEWS.rst:1]: … how long has that been there? :)

**Review by glyph — APPROVED** [review]
LGTM.

File: NEWS.rst
Comment (adiroiban) [reply NEWS.rst:1]: I will let the historian to write a story about it :p

thanks for the review.

I will trigger the final release

**Comment by adiroiban** [comment]
Final release triggered via GitHub Releases - build at https://github.com/twisted/twisted/actions/runs/3010698688

**Comment by adiroiban** [comment]
Note that towncrier was failing for this PR as the branch has no news fragment.

```
NEWS.rst
src/twisted/_version.py
----
No newsfragment found. Have you committed it?
ERROR: InvocationError for command /home/runner/work/twisted/twisted/.tox/newsfragment/bin/python bin/admin/check-newsfragment . (exited with code 1)
```

----------

In the past to had the PR merge, I would manually disabled the towncrier check inside the branch protection and then merge and then re-enable branch protection.

But now we have a newer towncrier version, that should know how to handle a release branch.

So I have updated the towncrier version in this PR.

To make it green

**Comment by adiroiban** [comment]
Auto-merge enabled.

**Pull request merged**
